lib/qmp.ml

```diff
@@ -282,6 +331,9 @@ let json_of_message = function
       | Add_fd id -> "add-fd", (match id with None -> [] | Some x -> [ "fdset-id", `Int x ])
       | Remove_fd id -> "remove-fd", ["fdset-id", `Int id]
       | Blockdev_change_medium (device, filename) -> "blockdev-change-medium", ["device", `String device; "filename", `String filename ]
+      | Device_del id -> "device_del", [ "id", `String id ]
+      | Device_add (driver, id, params) -> "device_add", (match params with None -> [ "driver", `String driver; "id", `String id ] | Some (x, y, z) -> [ "driver", `String driver; "id", `String id; "bus", `String x; "hostbus", `String y; "hostport", `String z ])
```

Comment:
Please split this in multiple lines, the other matches are in a line because the are compact:
```ocaml
      | Device_add (driver, id, params) -> "device_add", (
        match params with
        | None -> [ "driver", `String driver; "id", `String id ]
        | Some (x, y, z) -> [ "driver", `String driver; "id", `String id; "bus", `String x; "hostbus", `String y; "hostport", `String z ]
      )
```

Reply:
updated.